public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -361,11 +361,7 @@ export class LokiDatasource
       return this.runLiveQueryThroughBackend(fixedRequest);
     }
 
-    if (
-      config.featureToggles.lokiShardSplitting &&
-      requestSupportsSharding(fixedRequest.targets) &&
-      fixedRequest.app === CoreApp.Explore
```

Comment:
does this mean sharding will fire in dashboard panels now?

Reply:
Yes, that was the request, but only if the feature flag is enabled.

Reply:
Sounds good, wasn't sure as the PR description doesn't mention that

Reply:
Totally. Good call.

Reply:
Should be very similar to time splitting, but did we do a quick sanity check how it behaves with transformations?

Reply:
Good question. Yeah, same as time splitting.